src/main/java/core/mvc/tobe/NewDispatcherServlet.java

```diff
@@ -0,0 +1,47 @@
+package core.mvc.tobe;
+
+import core.mvc.ModelAndView;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class NewDispatcherServlet extends AbstractDispatcherServlet {
+    private static final Logger logger = LoggerFactory.getLogger(NewDispatcherServlet.class);
+    private static final String DEFAULT_REDIRECT_PREFIX = "redirect:";
+
+    private AnnotationHandlerMapping annotationHandlerMapping;
+
+    @Override
+    public void init() throws ServletException {
+        annotationHandlerMapping = new AnnotationHandlerMapping("next.controller");
+        annotationHandlerMapping.initialize();
+    }
+
+    @Override
+    protected void service(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        HandlerExecution handler = annotationHandlerMapping.getHandler(req);
```

Comment:
```suggestion
        String requestUri = req.getRequestURI();
        logger.debug("Method : {}, Request URI : {}", req.getMethod(), requestUri);
        HandlerExecution handler = annotationHandlerMapping.getHandler(req);
```
로그를 활용해서 어떤 것들이 등록이 되었는지 보는 것도 좋을 것 같아요 :)